infrastructure/helm-chart/charts/ingress-controller/templates/letsencrypt.yaml

```diff
@@ -0,0 +1,116 @@
+{{ if .Values.global.ingress }}
+{{ if .Values.global.ingress.https }}
+{{ if .Values.global.ingress.letsencryptEmail }}
+apiVersion: batch/v1
```

Comment:
This job is responsible for installing the `letsencrypt` resources. Instead of keeping the charts in the repo, we install them from a maintained location, using `kubectl apply`.